src/UID2.Client/KeyContainer.cs

```diff
@@ -12,6 +13,9 @@ internal class KeyContainer
         private readonly Dictionary<int, List<Key>> _keysBySite = new Dictionary<int, List<Key>>(); //for legacy /key/latest
 
         private readonly Dictionary<int, List<Key>> _keysByKeyset = new Dictionary<int, List<Key>>();
+
+        private readonly Dictionary<int, Site> _sites = new Dictionary<int, Site>();
```

Comment:
we've been using keyToValue naming conventions for maps, eg siteIdToSite

Reply:
renamed to _siteIdToSite